haskell-src/Concordium/Types/ProtocolVersion.hs

```diff
@@ -794,3 +795,17 @@ supportsEncryptedTransfers = \case
     SP7 -> False
     SP8 -> False
     SP9 -> False
+
+-- | Whether the protocol version supports Protocol Level Tokens (PLT).
+-- (Supported in 'P9' and onwards.)
+supportsProtocolLevelTokens :: SProtocolVersion pv -> Bool
+supportsProtocolLevelTokens = \case
```

Comment:
I already introduced `supportsPLT`. We should align the naming.

Reply:
anyway, I'll adjust this to fit with the naming currently used.